### DISCUSSION TITLE
自动运行后csgo显示正在游玩

### DISCUSSION THREAD

**Discussion by EvianRoit** [BeyondDimension/SteamTools#1142] [Q&A]
开机后用steam++打开steam，显示csgo正在游玩且无法点停止游戏.需要重新切换账号登录steam。现在正在用的软件版本是2.96 且上一个版本没有这个问题 但是之前的版本就有这个问题。 请问有同样问题的朋友吗 或者怎么解决
谢谢

**Comment by rmbadmin** [comment]
打开库存游戏 挂时长功能将CSGO移除出挂时长列表

**Reply by EvianRoit** [reply]
您好 谢谢回复 经过确认，我并没有把csgo放进挂时长列表里 我没有用过这个功能 所以我也觉得很奇怪

**Reply by rmbadmin** [reply]
> 您好 谢谢回复 经过确认，我并没有把csgo放进挂时长列表里 我没有用过这个功能 所以我也觉得很奇怪

先确认一下是不是有误操作过导致将CSGO添加进了挂时长列表，有可能无意中点到了

**Reply by EvianRoit** [reply]
ah 我去试一下 谢谢！
